package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ux-capture",
-  "version": "1.0.8",
+  "version": "1.1.0",
```

Comment:
This looks like it might be a breaking change - i.e., clients are going to have to update their library usage. I'd make this a major version bumpg - `2.0.0`

Reply:
I wonder if it's even worth making this change now then, because the only breaking part is renaming `label` to `name` in input structure.